src/+otp/+utils/Solver.m

```diff
@@ -1,12 +1,12 @@
 classdef Solver
     properties (Constant)
         Nonstiff = @ode45        
-        Stiff = otp.utils.compatibility.isOctave(@ode23s, @ode15s) % OCTAVE BUG: ode15s does not
+        Stiff = otp.utils.compatibility.isOctave(@ode23s, @ode15s) % OCTAVE FIX: ode15s does not
```

Comment:
Can I use something similar to fix QG pagemtimes?

Reply:
Yes. Could even have a separate f function. Unless it's a simple fix, I don't think it's necessary to fix in this PR

Reply:
Not this PR, but I am downloading octave to test for QG